src/models/document/document.ts

```diff
@@ -262,6 +262,14 @@ export const DocumentModel = types
     setProperties(properties: ISetProperties) {
       forEach(properties, (value, key) => self.setProperty(key, value));
     }
+  }))
+  .actions(self => ({
+    undoLastAction() {
+      alert("This is the undo action");
+    },
+    redoLastAction() {
+      alert("This is the redo action");
+    },
```

Comment:
@scytacki Out of curiosity, is this where we expect the undo/redo functionality to live or will it be in a separate entity (UndoManager?)?

Reply:
It'll probably be in an UndoManager, that is how it is implemented in the existing PR. The undo manager will be closely tired to the document so adding these actions temporarily to the documents seems like a good first step.